packages/expo-random/android/build.gradle

```
@@ -48,14 +48,6 @@ android {
   }
 }
 
-if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
-  apply from: project(":unimodules-core").file("../unimodules-core.gradle")
-} else {
-  throw new GradleException(
-      "'unimodules-core.gradle' was not found in the usual Flutter or React Native dependency locations. " +
-          "This package can only be used in such projects. Are you sure you've installed the dependencies properly?")
-}
-
 dependencies {
-  unimodule "unimodules-core"
+  implementation project(path: ':ReactAndroid')
```

```suggestion
  //noinspection GradleDynamicVersion
  implementation 'com.facebook.react:react-native:+'
```
We should depend on a compiled dependency of RN, otherwise people installing this module in their projects would need to have `:ReactAndroid` linked too. 

https://github.com/expo/expo/blob/5d1121fee4fc6522d9ee4cfabc25457bd5718c85/packages/expo-splash-screen/android/build.gradle#L73-L74

When developing Expo SDK a `resolutionStrategy` makes sure that libraries end up depending on `:ReactAndroid` instead of `…:react-native`

https://github.com/expo/expo/blob/2770008129aef794dcdf5fcdaa35ef5056b6ff29/android/app/build.gradle#L90-L92

---

ah thank you for pointing this out!